packages/cli/vitest.config.ts

```diff
@@ -12,13 +12,16 @@ import * as path from 'node:path';
 const __dirname = path.dirname(fileURLToPath(import.meta.url));
 
 export default defineConfig({
+  resolve: {
+    conditions: ['test'],
+  },
   test: {
-    include: ['**/*.{test,spec}.?(c|m)[jt]s?(x)', 'config.test.ts'],
+    include: ['**/*.{test,spec}.{js,ts,jsx,tsx}', 'config.test.ts'],
```

Comment:
this pattern was breaking running tests with the vitest plugin. not sure how this ever worked.